doc/sr_integration.rst

```diff
@@ -177,27 +177,34 @@ to the orchestrator. ::
 Following the guidelines from above, the first step is to initialize the client
 and create a unique identifier for the given processor. This should be done
 within roughly the same portion of the code where the rest of the model
-performs the initialization of other components. ::
+performs the initialization of other components.  ::
 
     ! Import SmartRedis modules
     use, only smartredis_client : client_type
+    ! Include all fortran enumerators especially for error checking
+    include "enum_fortran.inc"
 
     ! Declare a new variable called client and a string to create a unique
     ! name for names
     type(client_type) :: smartredis_client
     character(len=7)  :: name_prefix
-    integer :: mpi_rank, mpi_code, smartredis_code
+    integer :: mpi_rank, mpi_code, return_code
 
     ! Note adding use_cluster as an additional runtime argument for SmartRedis
     call initialize_model(temperature, number_of_timesteps, use_cluster)
-    call smartredis_client%initialize(use_cluster)
+    return_code = smartredis_client%initialize(use_cluster)
+    if (return_code /= SRNoError) stop 'Error in init'
```

Comment:
is there a reason to prefer /= vs .ne.?

Reply:
Not really; they are synonyms with each other. I changed these to `.ne.` for consistency